src/Bash/Simplify.hs

```diff
@@ -74,18 +74,50 @@ replace' somethingElse = return somethingElse
 
 replaceStr :: BashString -> State Namespace BashString
 replaceStr s@(SingleQ _) = return s
-replaceStr (DoubleQ s)   = DoubleQ `fmap` replaceStr' s
-replaceStr (NoQuote s)   = NoQuote `fmap` replaceStr' s
+replaceStr (DoubleQ s)   = expandStr s >>= \strs -> return $ DoubleQ $ map Right strs
```

Comment:
`expandStr s >>= (return . DoubleQ . map Right)`
Another style question, @fosskers: `return` or `pure`? I'd imagine `pure` due to AMP.